2020/11/19 13:13:51 [error] 6#6: *1 upstream timed out (110: Operation timed out) while reading response header from upstream, client: 172.31.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.31.0.2:9001", host: "symfony.localhost"
2020/11/19 13:23:31 [error] 6#6: *5 upstream timed out (110: Operation timed out) while reading response header from upstream, client: 172.19.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9001", host: "symfony.localhost"
2020/11/19 14:12:40 [error] 8#8: *5 FastCGI sent in stderr: "PHP message: 2020-11-19T14:12:40+00:00 [critical] Uncaught Exception: Failed to remove directory "/var/www/symfony/var/cache/dev/ContainerZrgB48O": rmdir(/var/www/symfony/var/cache/dev/ContainerZrgB48O): Directory not empty" while reading upstream, client: 172.22.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.2:9001", host: "symfony.localhost"
2020/11/19 15:34:44 [error] 6#6: *45 FastCGI sent in stderr: "PHP message: 2020-11-19T15:34:44+00:00 [critical] Uncaught Exception: Failed to remove directory "/var/www/symfony/var/cache/dev/ContainerUSMzOX2": rmdir(/var/www/symfony/var/cache/dev/ContainerUSMzOX2): Directory not empty" while reading upstream, client: 172.25.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.25.0.2:9001", host: "symfony.localhost"
2020/11/19 15:36:44 [crit] 6#6: *48 open() "/var/lib/nginx/tmp/fastcgi/1/00/0000000001" failed (13: Permission denied) while reading upstream, client: 172.25.0.1, server: symfony.localhost, request: "GET /user/new HTTP/1.1", upstream: "fastcgi://172.25.0.2:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/"
2020/11/26 20:34:29 [error] 7#7: *7 upstream timed out (110: Operation timed out) while reading response header from upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.2:9001", host: "symfony.localhost"
